packages/core/examples/textExample.tsx

```diff
@@ -25,9 +25,7 @@ export class TextExample extends BaseExample<ITextExampleState> {
                  "ellipsize to true",
     };
 
-    private onInputChange = handleStringChange((value: string) => {
-        this.setState({ textContent: value });
-    });
+    private onInputChange = handleStringChange((value: string) => this.setState({ textContent: value }));
```

Comment:
oh man there's one more step here 🎉 
call the param `textContent` and do the object shorthand `{ textContent }`